sale_product_multi_add/wizards/sale_import_products.py

```diff
@@ -46,7 +46,10 @@ def _get_line_values(self, sale, item):
             }
         )
         line_values = sale_line._convert_to_write(sale_line._cache)
-        line_values = sol_obj.play_onchanges(line_values, [])
+        line_values = sol_obj.play_onchanges(
+            line_values,
+            ["product_id", "price_unit", "product_uom", "product_uom_qty", "tax_id"],
```

Comment:
@majouda Thanks for this.

In fact I think the good syntax is to use `line_values.keys()` on the second argument of play_onchanges() to be compatible with side modules.

Could you maybe add a test case for the one you describe ?

Thanks